src/main/java/com/manardenza/login/CurrentUser.java

```diff
@@ -0,0 +1,21 @@
+package com.manardenza.login;
+
+import com.manardenza.entity.User;
+import lombok.Data;
+
+@Data
+public class CurrentUser {
+
+    private static CurrentUser loggedUser;
+    private User user;
+
+    public static CurrentUser getInstance() {
+        if (loggedUser == null) {
+            loggedUser = new CurrentUser();
+        }
+        return loggedUser;
+    }
+
+    private CurrentUser() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)